src/metabase/lib/js.cljs

```diff
@@ -347,6 +347,16 @@
                   key-str)]
     (u/->camelCaseEn key-str)))
 
+(defn- js-key->cljs-key
+  "Converts idiomatic JavaScript keys (`\"camelCaseStrings\"`) into idiomatic Clojure keys (`:kebab-case-keywords`).
+
+  A `\"is\"` prefix in JavaScript is replaced with a `?` suffix in Clojure , eg. `isManyPks` becomes `:many-pks?`."
+  [js-key]
+  (let [key-str (if (str/starts-with? js-key "is")
+                  (str (str/replace js-key #"^is" "") "?")
```

Comment:
I realize this is the inverse of the above function, but let's not inherit its bad habit - we already know the string starts with `is`, so let's do a cheap substring rather than use a regex replace.
```suggestion
                  (str (subs js-key 2) "?")
```